### PR TITLE
chore(README): cli options for qa

### PR DESCRIPTION
**Pull request by tooshel**
### What this PR does 📖
- Just updates the readme with instructions that I often need to go look up myself

### Which issue(s) this PR fixes 🔨
- Nothing! I didn't make a ticket since that is more overhead than just updating some text!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,7 +8,14 @@ If you do not have protobuf installed you can install it using homebrew: `brew i
 
 # Running
 
-To run the app in dev mode simply run `cargo run`
+To run the app in dev mode simply run `cargo run`. 
+
+The executable has a few command line options that might come in handy:
+ - `--path` which changes the folder used for storage, currently it's .cache but later it will go somewher else
```
Comment (stavares843) [inline README.md]: somewhere

File: README.md [hunk 2]
```diff
@@ -8,7 +8,14 @@ If you do not have protobuf installed you can install it using homebrew: `brew i
...
+ - `--title` which comes in handy if you have more than one window open
+
+From cargo you might run  `cargo run -- --title "User 1" --path .user1 ` and on some operating systmes you can run with different options in two seperate terminals
```
Comment (stavares843) [inline README.md]: separate

**Review by stavares843 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (tooshel) [reply README.md]: thanks! Fixed

File: README.md (hunk 2 above)
Comment (tooshel) [reply README.md]: thanks! Fixed

**Review by stavares843 — APPROVED** [review]

**Pull request merged**